building-view-counter/index040.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<!--
+    Now we see that the span tag is not displayed.
```

Comment:
Now we see that the `<span>` element is not displayed.
To display the view count with the current code, we have to get all of the text in the `<span>` and append the count to the end.
It would be easier if we can populate a field for the count right away.
To do this, add another `<span>` element at the end of the one you just added, and give it the `id` "viewCount".